exporter/templates_test.go

```diff
@@ -52,18 +52,9 @@ func xmlCodeRootNode(codeDisplay models.CodeDisplay) *xml.ElementNode {
 }
 
 func TestReasonTemplate(t *testing.T) {
-	// do not negate reason, r2 compatable
+	// do not negate reason
 	reason := models.CodedConcept{Code: "REASON_CODE_1", CodeSystem: "2.16.840.1.113883.6.1"} // specified in cms9_26.json
 	rootNode := xmlReasonRootNode(reason, false, true)
-	assertXPath(t, rootNode, "//entryRelationship", map[string]string{"typeCode": "RSON"}, nil)
```

Comment:
Not quite yet. It means probably just adding back in a few of those tests.

@ahubley , if I were you, I'd just add back in the change on (previously) lines 59 and 64, testing for the fields in //entryRelationship/observation. That way, there's one test for static information, and one for dynamic information.